contrib/uservices/badge/src/configuration.rs

```diff
@@ -0,0 +1,125 @@
+use config::{Config, ConfigError, Environment, File};
+use sdk_cds::service::APIConfiguration;
+
+#[derive(Default, Debug, Deserialize, Clone)]
+#[serde(default)]
+pub struct Configuration {
+    pub badge: BadgeConfiguration,
+}
+
+#[derive(Default, Debug, Deserialize, Serialize, Clone)]
+#[serde(default)]
+pub struct BadgeConfiguration {
+    pub url: String,
```

Comment:
You should use [url::Url](https://crates.io/crates/url) with the feature **serde** to load and implicit check the given url